### PR TITLE
182194044_remove_data_center_key_from_score

### PR DESCRIPTION
**Pull request by radek-polcode**
#### What's this PR do?
- USe data_center_key from data_centers

#### How should this be manually tested?
- Run tests
- Search for validators by data_center_key
- Run this branch on staging, check results after one day

#### What are the relevant tickets?
- https://www.pivotaltracker.com/story/show/182194044

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?

### CODE REVIEW

File: test/queries/validator_search_query_test.rb [hunk 1]
```diff
@@ -1,81 +1,95 @@
...
-      create(:validator, :with_score, account: 'Name1Account'),
-      create(:validator, :with_score, account: 'Name2Account'),
-      create(:validator, :with_score, name: 'Name1', network: 'searchnet'),
-      create(:validator, :with_score, name: 'Name2', network: 'searchnet'),
-      create(:validator, :with_score, account: 'Name1Account', network: 'searchnet'),
-      create(:validator, :with_score, account: 'Name2Account', network: 'searchnet'),
-      create(:validator)
+    # Setup data centers with data center hosts
+    data_center_name1 = create(:data_center, :china)
+    data_center_name1.update_column(:data_center_key, "Name1") # Update to match query
```
Comment (maciejkocylapc) [inline test/queries/validator_search_query_test.rb:10]: Can it be done in create?

File: test/queries/validator_search_query_test.rb [hunk 2]
```diff
@@ -1,81 +1,95 @@
...
+    data_center2 = create(:data_center, :berlin)
+
+    data_center_host = create(:data_center_host, data_center: data_center_name1)
```
Comment (maciejkocylapc) [inline test/queries/validator_search_query_test.rb:13]:
```suggestion
    data_center_host1 = create(:data_center_host, data_center: data_center_name1)
```

**Review by maciejkocylapc — CHANGES_REQUESTED** [review]

File: test/queries/validator_search_query_test.rb (hunk 1 above)
Comment (radek-polcode) [reply test/queries/validator_search_query_test.rb:10]: data_center_key is created in the before_save callback, so it's not what I needed

File: test/queries/validator_search_query_test.rb (hunk 2 above)
Comment (radek-polcode) [reply test/queries/validator_search_query_test.rb]: I changed name to `data_center`

**Pull request merged**